data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+		"topic": "BGSS_EVACUEE_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Evacuee_2" } 
+	},
+	{ "text": "Tell me about how you survived (evac3).", 
+		"topic": "BGSS_EVACUEE_3_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Evacuee_3" } 
+	},
+	{ "text": "Tell me about how you survived (evac4).", 
+		"topic": "BGSS_EVACUEE_4_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Evacuee_4" } 
+	},
+	{ "text": "Tell me about how you survived (evac5).", 
+		"topic": "BGSS_EVACUEE_5_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Evacuee_5" } 
+	},
+	{ "text": "Tell me about how you survived (evac6).", 
+		"topic": "BGSS_EVACUEE_6_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Evacuee_6" } 
+	},
+	{ "text": "Tell me about how you survived (evac7).", 
+		"topic": "BGSS_FEMA_EVACUEE_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_FEMA_Evacuee_1" } 
+	},
+	{ "text": "Tell me about how you survived (lfd1).", 
+		"topic": "BGSS_LEFT_FOR_DEAD_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Left_for_Dead_1" } 
+	},
+	{ "text": "Tell me about how you survived (lfd2).", 
+		"topic": "BGSS_LEFT_FOR_DEAD_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Left_for_Dead_2" } 
+	},
+	{ "text": "Tell me about how you survived (gungho1).", 
+		"topic": "BGSS_GUNG_HO_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Gung_Ho_1" } 
+	},
+	{ "text": "Tell me about how you survived (gungho2).", 
+		"topic": "BGSS_GUNG_HO_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Gung_Ho_2" } 
+	},
+	{ "text": "Tell me about how you survived (gungho3).", 
+		"topic": "BGSS_GUNG_HO_3_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Gung_Ho_3" } 
+	},
+	{ "text": "Tell me about how you survived (Prep1).", 
+		"topic": "BGSS_PREPPER_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Prepper_1" } 
+	},
+	{ "text": "Tell me about how you survived (prepper2).", 
+		"topic": "BGSS_PREPPER_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Prepper_2" } 
+	},
+	{ "text": "Tell me about how you survived (OutTown1).", 
+		"topic": "BGSS_OUT_OF_TOWN_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Out_of_Town_1" } 
+	},
+	{ "text": "Tell me about how you survived (OutTown2).", 
+		"topic": "BGSS_OUT_OF_TOWN_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Out_of_Town_2" } 
+	},
+	{ "text": "Tell me about how you survived (LostPart1).", 
+		"topic": "BGSS_LOST_PARTNER_1_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Lost_Partner_1" } 
+	},
+	{ "text": "Tell me about how you survived (LostPart2).", 
+		"topic": "BGSS_LOST_PARTNER_2_STORY1", 
+		"condition": { "npc_has_trait": "BGSS_Lost_Partner_2" } 
+	}
+
+
+
+
+
+
+
+
+     ]
+},
+{
+    "id": "BGSS_CONFUSED_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "I don't even fucking know anymore. I have no fucking idea what is going on. I'm just doing what I can to stay alive. World ended and I bungled along not dying, until I met you.",
```

Comment:
Replace fucking/shitty/etc with <swear> for more variety.

Reply:
Can I use a snippet in the middle of a string in a spot like this? I'd like to make these swears into snippets so that later on I can have the NPC personality alter what swears they choose...

Reply:
Yes, you can.